draft-vvv-webtransport-quic.md

```diff
@@ -129,23 +131,70 @@ the client receives a TLS Finished message from the server.
 
 In order to identify itself as a WebTransport application, QuicTransport relies
 on TLS Application-Layer Protocol Negotiation {{!RFC7301}}.  The user agent MUST
-request the ALPN value of "wq" and it MUST NOT establish the session unless that
-value is accepted.
+request the ALPN value of "wq-draft01" and it MUST NOT establish the session
+unless that value is accepted.
+
+## Client Indication
+
+In order to verify that the client's origin is allowed to connect to the server
+in question, the user agent has to communicate the origin to the server.  This
+is accomplished by sending a special handshake message, called *client
+indication*, on stream 2, which is the first client-initiated unidirectional
+stream.
+
+The client indication is a sequence of key-value pairs that are formatted in the
```

Comment:
We probably need a way to convey the end of the sequence. For example
- number of pairs up front
- total length up front
- FIN sent after the last one

Reply:
It's meant to be FIN.  Spelled this out more clearly.